[performance] remove duplicate print sinks

NicoK opened the pull request:
- alerts can be printed (as JSON) by setting alerts-sink to STDOUT
- currentRulesJson can be printed by setting rules-export-sink to STDOUT

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/ververica/lab-fraud-detection/2)
<!-- Reviewable:end -->

NicoK committed:
[performance] remove duplicate print sinks

- alerts can be printed (as JSON) by setting alerts-sink to STDOUT
- currentRulesJson can be printed by setting rules-export-sink to STDOUT

afedulov reviewed (changes requested):
I'd like to be able to see the alerts both in the IDE console output and affecting the web UI during the development. This change prevents it.

NicoK commented:
Both is possible by setting the sink to STDOUT. If this is about the results being JSON instead of a simple string representation in that case, that should be doable by refactoring `AlertsSink.createAlertsSink` to only convert to JSON when needed.

Irrespective of that, this can't be the way for production code. I do understand the need for printouts as some easy way of doing debugging / tests, but these lines should at least be commented out and only used during development. A debug/trace logger may be a more elegant way of getting what you wanted and making the outputs configurable. Given that you already have the stdout sink, I don't think there's much need for it though, or is it?

NicoK committed:
[debugging] allow JsonSerializer to trace-log inputs on behalf of its user

This allows having the alerts and current rules printed to the console again by
configuring trace logging for:
- com.ververica.field.dynamicrules.sinks.AlertsSink
- com.ververica.field.dynamicrules.sinks.CurrentRulesSink
